changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## `v0.16.0`
+
+### Features:
+
+- Added ability to pass a genesis file and peer address to `starport network chain join`
+- Added ability to show genesis or peer list with a flag in `starport network chain show`
+- `protoc` is now bundled with Starport CLI
+- Starport is now published automatically on the Docker Hub
```

Comment:
```suggestion
- Starport is now published automatically on the Docker Hub
```

Reply:
I remember hearing about this, what does this mean? was it not published before? or only manually published? what changes are required to see/manage this?

Reply:
Every time we merge changes into `develop`, image of Starport is published on Docker Hub automatically (this wasn't the case before). "How to" on this is something we need to include in the docs: https://github.com/tendermint/starport/issues/1017